Code/Editor/TrackView/AtomOutputFrameCapture.cpp

```diff
@@ -94,16 +94,24 @@ namespace TrackView
         m_captureFinishedCallback = AZStd::move(captureFinishedCallback);
 
         // note: "Output" (slot name) maps to MainPipeline.pass CopyToSwapChain
-        AZ::Render::FrameCaptureId frameCaptureId = AZ::Render::InvalidFrameCaptureId;
+        AZ::Outcome<AZ::Render::FrameCaptureId, AZ::Render::FrameCaptureError> captureOutcome;
```

Comment:
nit: extra long line, probably worth a typedef or using to give it a more descriptive name.

Reply:
I'll define a type for it.